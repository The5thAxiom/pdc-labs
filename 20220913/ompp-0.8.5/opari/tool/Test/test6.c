#include <omp.h>

int main() {
  // omp_init_lock(i)
  /* -- omp_init_lock(i) -- */
  "omp_init_lock(i)"  // omp_init_lock(i)
  "omp_init_lock(i)\"test" "\"omp_init_lock(i)" "omp_init_lock(i)\""
  """" "\"""\""

  omp_init_lock(i);       // omp_init_lock(i)
  omp_init_lock(i); omp_set_lock(i); omp_destroy_lock(i);
  printf("omp_init_lock(i)"); omp_init_lock(i);
  omp_init_locks(i);
}
