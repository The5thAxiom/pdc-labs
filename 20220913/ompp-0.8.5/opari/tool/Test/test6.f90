      program test6

      ! call omp_init_lock(i)
C f77 call omp_init_lock(i)
      "omp_init_lock(i)"  'omp_init_lock(i)'    ! call omp_init_lock(i)
      "omp_init_lock(i)""test" """omp_init_lock(i)" "omp_init_lock(i)"""
      """" """"""""

      "omp_init_lock(i)
C        "
     +omp_init_lock(i)"

      "omp_init_lock(i)    &
     !   "
      &omp_init_lock(i)"  call omp_init_lock(i)

      call omp_init_lock(i)       ! call omp_init_lock(i)
      call omp_init_lock(i) ; call omp_set_lock(i) ; call omp_destroy_lock(i)
      write(*,*) "call omp_init_lock(i)" ; call omp_init_lock(i)

C$C99 call omp_init_lock(i)
    !$DOMP  call omp_init_lock(i)

*$    call omp_init_lock(i)
*P$   call omp_init_lock(i)
    !$  call omp_init_lock(i)
    !P$ call omp_init_lock(i)

      end
