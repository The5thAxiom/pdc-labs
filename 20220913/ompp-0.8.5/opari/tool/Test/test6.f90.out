#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#line 1 "test6.f90"
      program test6

      ! call omp_init_lock(i)
C f77 call omp_init_lock(i)
      "omp_init_lock(i)"  'omp_init_lock(i)'    ! call omp_init_lock(i)
      "omp_init_lock(i)""test" """omp_init_lock(i)" "omp_init_lock(i)"""
      """" """"""""

      "omp_init_lock(i)
C        "
     +omp_init_lock(i)"

      "omp_init_lock(i)    &
     !   "
      &omp_init_lock(i)"  call POMP_Init_lock(i)

      call POMP_Init_lock(i)       ! call omp_init_lock(i)
      call POMP_Init_lock(i) ; call POMP_Set_lock(i) ; call POMP_Destroy_lock(i)
      write(*,*) "call omp_init_lock(i)" ; call POMP_Init_lock(i)

C$C99 call omp_init_lock(i)
    !$DOMP  call omp_init_lock(i)

*$    call POMP_Init_lock(i)
      call POMP_Init_lock(i)
    !$  call POMP_Init_lock(i)
        call POMP_Init_lock(i)

      end
